backend/wmg/pipeline/summary_cubes/cell_count.py

```diff
@@ -27,17 +27,26 @@ def extract(corpus_path: str) -> pd.DataFrame:
     """
     get obs data from integrated corpus
     """
-    return tiledb.open(f"{corpus_path}/{OBS_ARRAY_NAME}")
+    tiledb_array = tiledb.open(f"{corpus_path}/{OBS_ARRAY_NAME}")
+    return tiledb_array.df[:]
 
 
 def transform(obs: pd.DataFrame) -> pd.DataFrame:
     """
     Create cell count cube data frame by grouping data in the
     integrated corpus obs arrays on relevant features
     """
+
+    # filter out observations in the 'filter_cells' attribute.
+    # It is important to filter out rejected observations BEFORE
+    # performing the `groupby` operation because the `groupby` operation
+    # would lose information about the `filter_cells` attribute because
+    # `filter_cells` is not one of the columns used in the `groupby` list
+    cells_to_keep_bool_index = np.logical_not(obs["filter_cells"])
+    obs_to_keep = obs[cells_to_keep_bool_index]
```

Comment:
```suggestion
    obs_to_keep = obs[~obs["filter_cells"]]
```
This is a style thing, so not a blocker, but the above is more concise - `cells_to_keep_bool_index` is not being reused anywhere so no need for the intermediate variable.

Reply:
will fix